app/src/main/res/layout/manage_proguard.xml

```diff
@@ -154,21 +167,20 @@
                 <ImageView
                     android:layout_width="32dp"
                     android:layout_height="32dp"
-                    android:layout_marginRight="8dp"
+                    android:layout_marginLeft="8dp"
```

Comment:
why though? that makes the icons too close to the text imo and the previous layout looked better

Reply:
ok fixed